### PR TITLE
Add Home Screen And People Screen along with other components

### PR DESCRIPTION
**Pull request by beingnoble03**
**What kind of change does this PR introduce?**
Add Home Screen and People Screen along with other components

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**
image
image
Screenshot 2023-07-16 at 5 11 51 PM
Screenshot 2023-07-16 at 5 12 09 PM

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by beingnoble03** [comment]
The tests are failing due to unacceptable level of code coverage. Writing tests for the screens is the target for the next week.

File: src/screens/OrgPost/OrgPost.tsx [hunk 1]
```diff
@@ -232,131 +201,172 @@
...
+                        });
+                      }
+                    }}
+                    data-testid="organisationVideo"
+                  />
+                </label>
+                {postformState.postVideo && (
+                  <div className={styles.preview}>
+                    <video controls>
+                      <source src={postformState.postVideo} type="video/mp4" />
```
[the remark this answers is not in the document]
Comment (beingnoble03) [reply src/screens/OrgPost/OrgPost.tsx:341]: This file isn't changed by me. This change was brought up in this PR. https://github.com/PalisadoesFoundation/talawa-admin/pull/939

**Comment by evasharma12** [comment]
@beingnoble03 checks are failing in your PR. Look into them.

**Comment by beingnoble03** [comment]
@evasharma12 the check is failing not due to code changes but due to some improper configuration in Github actions. I think @kb-0311 is working on it. This was discussed in the group. Thanks.

**Comment by kb-0311** [comment]
@beingnoble03 The errors logged during the introspection checks are due to invalid queries defined in this repo as they do not exist on the backend.

@rishav-jha-mech had removed them except one and hence he is getting 1 error instead of 3 like this one 
https://github.com/PalisadoesFoundation/talawa-admin/pull/940#issuecomment-1630354248

**Comment by kb-0311** [comment]
@beingnoble03 I think you have misunderstood. The issue is not whether the queries are used in a component the problem is that these queries are not defined in the API. Essentially making those queries useless, we now know about it in a PR instead of at runtime.
You can go to the API schema definitions and you won't find those queries defined.

**Comment by rishav-jha-mech** [comment]
@kb-0311 yes, only one mutation is left rn for updateSpamNotification and it doesn't exist in the backend, I'll have to remove it for the test to pass ig

**Comment by beingnoble03** [comment]
@rishav-jha-mech Can you please open a separate PR for it? So that when that PR gets merged we can update our branches?

**Comment by rishav-jha-mech** [comment]
Opened an issue #945 , will soon open a PR for it and update this comment 😇

**Comment by beingnoble03** [comment]
I'll update this PR once https://github.com/PalisadoesFoundation/talawa-admin/pull/946 gets merged. This will fix the failing test.

**Comment by beingnoble03** [comment]
Thanks @rishav-jha-mech. I was able to fix that failing tests and sync with mutations with the current version of talawa-api.

**Comment by beingnoble03** [comment]
I have updated this PR with 4 weeks work now. This work was planned to be completed by July 26.

**Review by evasharma12 — APPROVED** [review]

**Comment by evasharma12** [comment]
@CyberWake @Sagar2366 kindly review this PR.

**Review by noman2002 — APPROVED** [review]
The changes looks good to me. 

Are we building the user portal inside the admin ? If yes, then how we are going to deploy it?

**Comment by beingnoble03** [comment]
@noman2002 we'll need to shift the routes of Talawa's admin portal to something like `www.xyz.com/admin/<extra_path_needed_for_talawa_admin_portal>` as for regular users the route should always on the root e.g. `www.xyz.com/<extra_path_needed_for_talawa_user_portal>`. But as of now, as the talawa-user-portal is under dev, we can keep the current routes.

**Review by evasharma12 — APPROVED** [review]

**Pull request merged**